packages/postcss-zindex/src/index.js

```diff
@@ -29,7 +29,7 @@ function pluginCreator(opts = {}) {
             return;
           }
 
-          cache.optimizeValues();
+          cache.optimizeValues(opts.startIndex || 1);
```

Comment:
The start index is only used inside `optimizeValues()` so I find passing it as an argument reduces indirection (instead of first storing it as an instance variable)